lib/src/v3/api/person.dart

```diff
@@ -18,6 +18,7 @@ class Login with _$Login implements LemmyApiQuery<LoginResponse> {
   const factory Login({
     required String usernameOrEmail,
     required String password,
+    required String totp2faToken,
```

Comment:
is this really required? Should be optional right? or does the API now require it even if TOTP isn't enabled?

Reply:
Fixed this and added a JsonKey override per our conversation in chat.